BLSQ/iaso
# IA-2402: OU change request improvement

Display comment while rejected, make instance links using the same colors (red, orange or green)

Related JIRA tickets : IA-2402

## Self proofreading checklist


## Changes

- comment component
- css to override links colors

## How to test

- open org unit change request page, open a rejected one and check that you can see the comment
- on approved and rejected changes, if a reference instance has been changed, the link should be red or green 

## Print screen / video
<img width="1175" alt="Screenshot 2024-01-11 at 12 50 17" src="https://github.com/BLSQ/iaso/assets/12494624/960d987f-21ba-4006-a795-175a46a12eb3">
<img width="1332" alt="Screenshot 2024-01-11 at 12 47 14" src="https://github.com/BLSQ/iaso/assets/12494624/db8b6dee-6191-4620-a770-6273dfae9198">
